apps/trace/models.py

```diff
@@ -0,0 +1,21 @@
+from django.db import models
+
+from apps.utils.models import BaseModel
+
+
+class Trace(BaseModel):
```

Comment:
This should have a 'team' field as well. Either it can extend `BaseTeamModel` or just `models.Model` and add `team` and `timestamp`. We don't really need the `updated_at` date and also don't need the `VersioningMixin` that comes with the other base models.